### ISSUE TITLE
Column Clarifications

### ISSUE THREAD

**Issue by MrinankSharma** [rearc-data/covid-19-deutschland-robert-koch-institut#3]
Hi, 

Thanks for making this dataset available. I wanted to clarify the meaning of some columns. In short, I'm looking for data from Germany which has the dates of symptom onset and the date the case was reported. 

My understanding that is if `IstErkrankungsbeginn ==1`, then `Refdatum` is the date of symptom onset. Would I be correct in understanding that `Meldedatum` is the date that the case was reported? 

Thanks!

**Comment by masinazarian** [comment]
Hi @MrinankSharma,

Thank you for your comment. You can refer to the dataset page here to see more detailed explanations of each column. [Here](https://npgeo-corona-npgeo-de.hub.arcgis.com/datasets/dd4580c810204019a7b8eb3e0b329dd6_0?selectedAttribute=Refdatum)'s a translated version of the ones you mentioned:

Referenzdatum (Refdatum): Date of illness or, if this is not known, the reporting date
IstErkrankungsbeginn: 1 if the reference date is the onset of illness, 0 otherwise
Meldedatum: Date when the health department became aware of the case

So basically Referenzdatum refers to either "date of symptom" (if IstErkrankungsbeginn == 1) or "date of reporting" (if IstErkrankungsbeginn == 0). 

Meldedatum is the same date as Referenzdatum in the latter case - this is my understanding looking into the data.  

Hope this helps.
